package.json

```diff
@@ -55,7 +55,7 @@
     "nodemon": "^3.0.3"
   },
   "scripts": {
-    "start": "node main.js",
+    "start": "BUILD=build WFINFO_FILTERED_ITEMS=$WFINFO_FILTERED_ITEMS WFINFO_PRICES=$WFINFO_PRICES node main.js",
```

Comment:
very hacky me not like

Reply:
well env vars are already in the shell env there shouldn't be a need to do this

Reply:
or am i missing something?

Reply:
even if it's duplicative.... why is that hacky? and `BUILD=build` isn't default in the shell (or in context, as i don't generally run these in shell)

Reply:
well BUILD=build is understandable but ` WFINFO_FILTERED_ITEMS=$WFINFO_FILTERED_ITEMS WFINFO_PRICES=$WFINFO_PRICES` this just seems that you're trying to fix some very specific node issue with the shell environment

Reply:
as far as my understanding of bash and powershell this really doesnt make sense that's why i called it hacky

Reply:
it's to parallel the working conditions of the build command.

Reply:
i see it's hacky then glad you agree with me